README.md

```diff
@@ -56,4 +56,9 @@ USAGE :
     </clientClasses>
   </configuration>
  </plugin>
-```
\ No newline at end of file
+```
+
+
+PUBLISHING TO GITHUB PACKAGES :
+-----------------------
+Before merging your pr, please make sure that you bump up the version in the pom file.
```

Comment:
suggestion: I'd scope this based on the action that someone is performing. If it were me, I could easily miss anything under "PUBLISHING TO GITHUB PACKAGES" if I'm not in the mindset that I need to publish anything. I'd also make this clear what happens if you don't update the version in the pom file.
```
MERGING PRs:
----------------
After each merge to main, the package is built and an attempt is made to release to the Github Java Repository. This only succeeds where the version does not already exist in the repo.

Please make sure the version is increased in the `pom.xml` file before every merge to main.
```

Reply:
updated, thanks.